automation/run-tests.sh

```diff
@@ -109,15 +122,17 @@ function run_tests {
         container_exec "
           cd $CONTAINER_WORKSPACE &&
           pytest \
-            --verbose --verbose \
-            --log-level=DEBUG \
-            --log-date-format='%Y-%m-%d %H:%M:%S' \
-            --log-format='%(asctime)s %(filename)s:%(lineno)d %(levelname)s %(message)s' \
-            --durations=5 \
-            --cov /usr/lib/python*/site-packages/libnmstate \
-            --cov /usr/lib/python*/site-packages/nmstatectl \
-            --cov-report=html:htmlcov-integ \
-            --cov-report=term \
+            $PYTEST_OPTIONS \
+            tests/integration \
+            ${nmstate_pytest_extra_args}"
+    fi
+    if [ $TEST_TYPE == $TEST_TYPE_ALL ] || \
+       [ $TEST_TYPE == $TEST_TYPE_INTEG_SLOW ];then
+        container_exec "
+          cd $CONTAINER_WORKSPACE &&
+          pytest \
+            $PYTEST_OPTIONS \
+            -m slow --runslow \
```

Comment:
It will be nice if `--runslow` will implicitly add the `-m slow`.
But this is not that important as I do not expect to run these tests often directly (bypassing the script). I usually do that for debugging.

Reply:
--runslow means slow test case can be included.

-m slow means run slow test case only.